woocommerce/woocommerce
# Add Media Uploader component

### All Submissions:






### Changes proposed in this Pull Request:

Adds a component for uploading media via button as well as drag and drop.

Closes # .

### How to test the changes in this Pull Request:

1. Run `npm run storybook`
2. Check out the various examples and examine media functionality (note that the WP media modal does not work here since we are outside the WP context)
3. Add the component to any WC React page with media scripts enqueued (should be most WCA pages).
```
<MediaUploader
    onSelect={ ( file ) => console.log( file ) }
    onError={ ( error ) =>console.log( error ) }
    onUpload={ ( files ) => console.log( files ) }
/>
```
4. Check that the WP media modal opens correctly and allows you to choose an image.

### Other information:




### FOR PR REVIEWER ONLY:
